### PR TITLE
[BLOG] Add python-library-function-usage

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-blog-name-quansight.vercel.app/blog/blog-name).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

**Comment by MarsBarLee** [comment]
@noatamir The Vercel build states `Error: There is not matching author in database` for christopher-ostrouchov. ([Link to build error](https://vercel.com/quansight/labs/F63DUtZW6eHLoQe2fkzGWLJPFBYF)).

I've checked the spelling, there are no accented characters and this author is has not been previously published in the updated Labs blog post.

**Comment by noatamir** [comment]
@pavithraes Can you check if `christopher-ostrouchov` appears as an author in Labs' StoryBlock, and if not add him? I no longer have access.

**Comment by noatamir** [comment]
LGTM (metadata and links are in order), waiting for author issue to be resolved, so the preview can be checked as well.

File: apps/labs/posts/python-library-function-usage.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
+---
+title: 'Measuring API usage for popular numerical and scientific libraries'
+published: May 27, 2019
+author: christopher-ostrouchov
```
Comment (pavithraes) [inline apps/labs/posts/python-library-function-usage.md:4]:
```suggestion
author: chris-ostrouchov
```

@noatamir His first name is saved as "Chris" in Storyblok. Let me know if I should update it to "Christopher", or if this suggestion works. :)

Comment (noatamir) [reply apps/labs/posts/python-library-function-usage.md:4]: @costrouc any preference for how you are named in the teams page AND labs blog?

**Review by noatamir — APPROVED** [review]
LGTM. merging

**Pull request merged**